index.html

```diff
@@ -1348,6 +1348,19 @@ <h2>
 MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
 property. Both the raw 32-byte base58-btc format and JWK format MUST be
 supported by DID Document Processors. The use of other key formats for RSA MUST
+cause the DID Document Processor to raise an error.
+            </td>
+          </tr>
+          <tr>
+            <td>
+ed25519
+            </td>
+            <td>
+ed25519 key values MAY be expressed as the raw 32-byte value encoded in
+base58-btc format using the <code>publicKeyBase58</code> property or
+MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
+property. Both the raw 32-byte base58-btc format and JWK format MUST be
+supported by DID Document Processors. The use of other key formats for RSA MUST
```

Comment:
This line has RSA instead of ed25519?

Reply:
This was a mistake, will fix.

Reply:
Fixed in 41b0b4781bab8fdfae2cc42990bacc670342e04f.